README.md

```diff
@@ -171,7 +206,7 @@ UIButton *settingsButton;
 UIButton *setBut;
 ```
 
-A two letter prefix should always be used for class names and constants, however may be omitted for Core Data entity names. For common AnchorFree entities prefix 'AF' should be used. Each project should use own prefix for custom entities (BN, HT etc.)
+A common, per project prefix should always be used for class names and constants. It may however be omitted for Core Data entity names. For common AnchorFree entities prefix 'AF' should be used. Each project should use its own prefix for custom entities (BN, HT etc.). New projects should use three letter prefixes as Apple reserves two letter ones for present and future APIs.
```

Comment:
Using two letters prefixes in our projects is a thing that baffles me every time I see these "AF", "BN", "HT", etc.. Especially "AF", which, as we all know, collides with [Alamofire](https://github.com/Alamofire/Alamofire).
Moreover, I've seen Apple has a private symbol with "AF" prefix somewhere inside a System SDKs ( `AFAnalytics`, if I'm not mistaken).

Reply:
Ideally we'd have a three letter prefix for our stuff but changing that policy seems overkill considering the amount of legacy code around. It may be worth doing but probably as a separate debate.

Reply:
Or just moving everything to Swift which has module-based namespacing 😉

Reply:
As a side note: we have few publicly available SDKs, where this prefix policy starts to play somewhat more important role...

Reply:
On that note, I kinda figured we could start using HSS as the prefix for Hotspot Shield related classes.